linux_docker_resources/Dockerfile

```diff
@@ -94,7 +94,7 @@ RUN if test \( ${UBUNTU_DISTRO} = jammy -a ${ROS_DISTRO} != humble \); then apt-
 # Iron uses python3-lttng, but then starting after Iron we switch to liblttng-ctl-dev.
 RUN if test \( ${UBUNTU_DISTRO} = jammy -a ${ROS_DISTRO} = iron \); then apt-get update && apt-get install --no-install-recommends -y \
   python3-lttng; fi
-RUN if test \( ${UBUNTU_DISTRO} = jammy -a ${ROS_DISTRO} = rolling \); then apt-get update && apt-get install --no-install-recommends -y \
+RUN if test \(\( ${UBUNTU_DISTRO} = jammy -o ${UBUNTU_DISTRO} = noble \) -a ${ROS_DISTRO} = rolling \); then apt-get update && apt-get install --no-install-recommends -y \
```

Comment:
After looking at this and thinking about it again, I think I steered you wrong.  In particular, we don't need the `UBUNTU_DISTRO` check at all, we can really just get by with the `ROS_DISTRO` check.  I'm going to push a new commit here that does that, along with simplifies a few of the above lines.